README.md

```diff
@@ -237,47 +237,6 @@ This section shows detailed information about a selected:
 
 ![Trace Viewer Item Properties](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-item-properties-0.0.2.png)
 
```

Comment:
Sentence in line 7 needs to be updated and point to the CONTRIBUTING.md file:
Read our [contributor guide](CONTIBUTING.md) and follow the instructions to contribute code.